app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -653,6 +654,10 @@ function AcuantCapture(
     });
   }
 
+  function onImageCaptureFeedback(text: string) {
```

Comment:
I keep waffling whether it's better to have this function, or remove it and just use `setImageCaptureText` 🤷

Reply:
I think it's six one way and a half a dozen the other. 🤷🏻‍♀️